### PR TITLE
Improve docs of the Apify Scrapy template

### LINKED ISSUE #228: Create documentation for Scrapy wrapper
- Improve the docs/readme of the Scrapy Actor template itself
    - add disclaimers/limitations of the template
    - add code notes - what should not be removed, where users should add their components, what files whey should update, ...
- Create a doc page at [docs.apify.com/cli/](https://docs.apify.com/cli/)

### PR DESCRIPTION
**Pull request by vdusek**
Closes #228

### CODE REVIEW

File: templates/python-scrapy/src/__main__.py [hunk 1]
```diff
@@ -1,9 +1,18 @@
...
+This module transforms a Scrapy project into an Apify Actor, handling the configuration of logging, patching Scrapy's
+logging system, and establishing the required environment to run the Scrapy spider within the Apify platform.
 
-# We need to configure the logging first before we import anything else,
-# so that nothing else imports `scrapy.utils.log` before we patch it.
+This file is specifically designed to be executed when the project is run as an Apify Actor using `apify run` locally
+or being run on the Apify platform. It is not being executed when running the project as a Scrapy project using
+`scrapy crawl`.
+
+We recommend do not modify this file unless you really know what you are doing.
```
Comment (barjin) [inline templates/python-scrapy/src/__main__.py:9]:
```suggestion
We recommend you do not modify this file unless you really know what you are doing.
```

File: templates/python-scrapy/src/main.py [hunk 2]
```diff
@@ -1,45 +1,105 @@
...
+
+Feel free to customize this file to add specific functionality to the Actor, such as incorporating your own Scrapy
+components like spiders and handling Actor input. However, make sure you have a clear understanding of your
+modifications. For instance, removing Apify-Scrapy components from the settings will break the integration
+between Scrapy and Apify.
+
+Known limitations to be aware of:
+---------------------------------
+
+1. Asynchronous spiders and Twisted & Asyncio integration
```
Comment (barjin) [inline templates/python-scrapy/src/main.py:18]:
```suggestion
1. Asynchronous spiders and Twisted & AsyncIO integration
```

File: templates/python-scrapy/src/main.py [hunk 3]
```diff
@@ -1,45 +1,105 @@
...
+
+   Asynchronous spiders (and possibly other components) may encounter challenges due to the Twisted & Asyncio
```
Comment (barjin) [inline templates/python-scrapy/src/main.py:20]:
```suggestion
   Asynchronous spiders (and possibly other components) may encounter challenges due to the Twisted & AsyncIO
```

**Review by barjin — APPROVED** [review]
Looking good, just some tiny nits :) Cheers!

**Comment by vdusek** [comment]
@barjin nice, thanks

**Pull request merged**